jenkins-design-language/less/components/button.less

```diff
@@ -129,6 +129,10 @@ button, .btn, .btn-default, .btn-primary, .btn-secondary, .btn-danger, a.btn-lin
 button, .btn, .btn-default, .btn-primary, .btn-secondary {
     &.monochrome {
         .btn-styles(transparent, @gray-light, @text-color, @gray-lighter, @gray-lighter);
+
+        svg {
+            fill: @text-color !important;
```

Comment:
I think I just opened a can of worms, a lot of icons have a bad inherited colour if we do it as I proposed. I don't think the benefit is worth it right now, if we have a different reason to remove this important rule or to add the fill colour by attr instead of inline style I will come back to it in a different PR, as it is, everything works as intended and I think we should merge as is.